File: Assignmentno17/task_4.java
import java.util.*;

class Logic
{
    void FindMin(int iNo1,int iNo2,int iNo3)
    {

        if(iNo1<iNo2&&iNo1<iNo3)
        {
            System.out.println("The Minimum number is :"+iNo1);
        }
        else if(iNo2<iNo1&&iNo2<iNo3 )
        {
             System.out.println("The Minimum number is :"+iNo2);

        }
        else 
        {
             System.out.println("The Minimum number is :"+iNo3);
        }

    }
}

class task_4
{
    public static void main(String A[])
    {
        int iValue1 = 0;
        int iValue2 = 0;
        int iValue3 = 0;

        Scanner sobj = new Scanner(System.in);
        Logic lobj = new Logic();

        System.out.println("Enter the first number:");
        iValue1=sobj.nextInt();

        System.out.println("Enter the second number:");
        iValue2=sobj.nextInt();

        System.out.println("Enter the third number:");
        iValue3=sobj.nextInt();

        lobj.FindMin(iValue1,iValue2,iValue3);




    }
}

File: Assignmentno6/task_4.c
#include<stdio.h>
int Multiplication(int iNo1,int iNo2,int iNo3)
{
    int iMul= 1;
    int flag = 0;

    if(iNo1 != 0)
    {
        iMul = iMul * iNo1;
        flag = 1;
    }
    if(iNo2 != 0)
    {
        iMul = iMul*iNo2;
        flag = 1;
    }
    if(iNo3 != 0)
    {
        iMul = iMul * iNo3;
        flag = 1;
    }
    if(flag == 0)
    {
        
        return 0;
    }
    return iMul;

}

int main()
{
    int iValue1 = 0, iValue2 =0,iValue3 = 0;
    int iRet = 0;

    printf("Enter they three numbers:\n");
    scanf("%d %d %d",&iValue1,&iValue2,&iValue3);

    iRet = Multiplication(iValue1,iValue2,iValue3);
    printf("The multiplication of three numbers is :%d\n",iRet);

    return 0;
}